eodag/plugins/apis/cds.py

```diff
@@ -421,3 +431,53 @@ def download_all(
             timeout=timeout,
             **kwargs,
         )
+
+    def discover_queryables(
+        self, product_type: Optional[str] = None, **kwargs: Optional[Dict[str, Any]]
+    ) -> Optional[Dict[str, Tuple[Annotated, Any]]]:
```

Comment:
- Annotated should be qualified type: Optional[Dict[str, Tuple[Annotated[Any, FieldInfo], Any]]]
- **kwargs type hint should be Any
- **kwargs missing in docstring